TrainingHW10/TrainingHW9/Сatalog.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+ namespace TrainingHW9
+{
+    public class Catalog
+    {
+        List<Book> _books;
```

Comment:
is this field private according to task?

Reply:
Yes